src/Terrain/PlanetWrenderer.h

```diff
@@ -187,6 +169,16 @@ class IcoSphereTree : public RefCounted
 private:
 
     //PODVector<PlanetWrenderer> m_viewers;
+    PODVector<float> m_vertBuf;
```

Comment:
Re-arranging things in this way is an attempt at improving the stuct packing of the class.